3scale/porta
# THREESCALE-11499: 3scale accounts listing can't filter by state

As exposed in [THREESCALE-11499](https://issues.redhat.com/browse/THREESCALE-11499), at Accounts > Listing it wasn't possible to filter `Created` accounts. This PR fixes that problem.

**Before**
![Captura de pantalla 2024-12-13 a las 11 24 29](https://github.com/user-attachments/assets/1cf13721-4a56-4307-9914-f173c1ca4d95)

**After**
![Captura de pantalla 2024-12-13 a las 11 25 04](https://github.com/user-attachments/assets/35db3163-7646-4e4d-893e-0aebf8777b8d)
![Captura de pantalla 2024-12-13 a las 11 23 49](https://github.com/user-attachments/assets/09836a67-c034-4764-aab1-8d505e0d3044)

**Verification steps** 
1. Go to Audience > Accounts > Listing
2. Add a new account
3. Confirm the State for that new account is Created
4. Use the State filter and select Created
5. Confirm the recently created (and other accounts which might be in State Created) appears
